sdks/java/testing/expansion-service/build.gradle

```diff
@@ -28,7 +28,7 @@ dependencies {
   compile project(path: ":runners:core-construction-java")
   compile project(path: ":sdks:java:io:parquet")
   compile project(path: ":sdks:java:core", configuration: "shadow")
-  provided library.java.hadoop_client
+  testRuntime library.java.hadoop_client
```

Comment:
Is this change related ?

Reply:
Yes. test expansion service jar is also used for SDK container dependency. hadoop client library should be included in the shadow jar.